locales/en.json

```diff
@@ -524,10 +524,10 @@
         "unsupported": "Your browser does not support push notifications.",
         "warning": {
           "enable_close": "Close",
-          "enable_description": "To receive notifications when Elk is not open, enable push notifications. You can control precisely what types of interactions generate push notifications via the \"@:settings.notifications.show_btn{'\"'} button above once enabled.",
-          "enable_description_desktop": "To receive notifications when Elk is not open, enable push notifications. You can control precisely what types of interactions generate push notifications in \"Settings > Notifications > Push notifications settings\" once enabled.",
+          "enable_description": "To receive notifications when Mozilla.social is not open, enable push notifications. You can control precisely what types of interactions generate push notifications via the \"@:settings.notifications.show_btn{'\"'} button above once enabled.",
```

Comment:
There's 3 different versions of this UI based on a prop called `closeableHeader` and a css check for `xl`